go/vt/vtgate/legacy_scatter_conn_test.go

```diff
@@ -531,7 +531,7 @@ func TestScatterConnSingleDB(t *testing.T) {
 	assert.Contains(t, errors[0].Error(), want)
 
 	// TransactionMode_MULTI in txconn. Should not fail.
-	sc.txConn.mode = vtgatepb.TransactionMode_MULTI
+	sc.txConn.txMode = &StaticConfig{TxMode: vtgatepb.TransactionMode_MULTI}
```

Comment:
@GuptaManan100 It looks like the default for `StaticConfig` for the DDL settings is different from the default in Vitess, should that always be true by default? We could achieve that by using an initializer function for it instead of a direct struct initialization?

Reply:
For now, its only being used for the txmode property, and that too mostly in tests (just 1 instancd of vtexplain_vtgate using it other than tests). I think it should be okay to specify them explicitly from the tests.